### PR TITLE
IDVA5-1257-Json Data Mapping 

### PR DESCRIPTION
**Pull request by atedros-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by cbroadley-ch — DISMISSED** [review]

File: src/main/java/uk/gov/companieshouse/acsp/service/FilingsService.java [hunk 1]
```diff
@@ -324,6 +324,9 @@ private Address buildRegisteredOfficeAddress(AcspDataDto acspDataDto) {
         registeredOfficeAddress.setRegion(
               Optional.ofNullable(acspDataDto.getRegisteredOfficeAddress().getRegion())
                       .map(String::toUpperCase).orElse(null));
+      registeredOfficeAddress.setLocality(
```
Comment (swalke-ch) [inline src/main/java/uk/gov/companieshouse/acsp/service/FilingsService.java:327]: Is setting locality not required in buildCorrespondenAddress method ?

Comment (atedros-ch) [reply src/main/java/uk/gov/companieshouse/acsp/service/FilingsService.java:327]: this a test fix for the mapping mismatch we got in the registered address xml generated file, if it works good and fix the issue I will update the corrAddress after

**Review by swalke-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**